server/request_processor.go

```diff
@@ -355,7 +357,9 @@ func (r *RpcRequest) GetAddressNonceRange(address string) (minNonce, maxNonce ui
 		r.logger.Error("[GetAddressNonceRange] eth_getTransactionCount marshal failed", "error", err)
 		return 0, 0, err
 	}
+	start := time.Now()
 	httpRes, err := r.client.ProxyRequest(jsonData)
+	r.logger.Info("[GetAddressNonceRange] after making ProxyRequest", "timeTaken", time.Since(start))
```

Comment:
Need this logging to know the exact timeTaken to call proxyd. We could remove this later if not required or keep it

Reply:
What not do this inside ProxyRequest?

Reply:
logger is not available inside proxy request client.

Reply:
updated